pkg/eks/components/Import.vue

```diff
@@ -0,0 +1,120 @@
+<script>
+import { _EDIT } from '@shell/config/query-params';
+import debounce from 'lodash/debounce';
+import LabeledSelect from '@shell/components/form/LabeledSelect.vue';
+import LabeledInput from '@components/Form/LabeledInput/LabeledInput.vue';
+import Checkbox from '@components/Form/Checkbox/Checkbox.vue';
+
+export default {
+  name: 'ImportEKS',
+
+  emits: ['update:clusterName', 'error', 'update:enableNetworkPolicy'],
+
+  components: {
+    LabeledSelect, LabeledInput, Checkbox
+  },
+
+  props: {
+    mode: {
+      type:    String,
+      default: _EDIT
+    },
+
+    // name of cluster to be imported
+    // this wont necessarily align with normanCluster.name as it would w/ provisioning
+    clusterName: {
+      type:    String,
+      default: ''
+    },
+
+    credential: {
+      type:    String,
+      default: null
+    },
+
+    region: {
+      type:    String,
+      default: ''
+    },
+
+    enableNetworkPolicy: {
+      type:    Boolean,
+      default: false
+    }
+  },
+
+  created() {
+    this.debouncedlistEKSClusters = debounce(this.listEKSClusters, 200);
+    this.debouncedlistEKSClusters();
+  },
+
+  data() {
+    return { loadingClusters: false, clusters: [] };
+  },
+
+  watch: {
+    region() {
+      this.debouncedlistEKSClusters();
+    },
+    cloudCredentialId() {
+      this.debouncedlistEKSClusters();
+    }
+  },
+
+  methods: {
+
+    async listEKSClusters() {
+      if (!this.region || !this.credential) {
+        return;
+      }
+      this.loadingClusters = true;
+      try {
+        const eksClient = await this.$store.dispatch('aws/eks', { region: this.region, cloudCredentialId: this.credential });
+
+        const res = await eksClient.listClusters({});
+        const clusters = res?.clusters;
+
+        if (!clusters) {
+          return;
+        }
+
+        this.clusters = res?.clusters;
+      } catch (err) {
+        this.$emit('error', err);
+      }
+
+      this.loadingClusters = false;
+    },
+  }
+};
+</script>
+
+<template>
+  <div class="row mb-10 align-center">
+    <div class="col span-6">
+      <LabeledSelect
```

Comment:
I think we want to make this a required field, it can't think of a scenario where we wouldn't want it. 

Without it we'll get an error:
![image](https://github.com/user-attachments/assets/2f209c67-47ad-490d-84ac-0fd172544cc7)